frontend/test/cypress/integration/product-list/collections_scroll.spec.ts

```diff
@@ -0,0 +1,27 @@
+// Assert that when a user clicks the pagination buttons, 
+// the page scrolls back up.
+
+describe('collections tools scroll up', () => {
+   const user = cy;
+   beforeEach(() => {
+      user.visit('/Tools');
+   });
+ 
+   it('Make sure product list scrolls up after clicking next page', () => {
+      user.findByTestId('collections-search-box').should('be.visible');
+      user.findByTestId('next-page').click();
+
+      // Wait for the next page to be updated and scrolled to top.
+      user.wait(3000);
+
+       // When it scrolls to the top, the search bar should be visible
+      user.window().isWithinViewport(user.get('[data-testid=collections-search-box]'));

```

Comment:
If we plan on creating more `data-*` selectors we may want to add another custom command like this https://docs.cypress.io/guides/references/best-practices#Real-World-Example

```js
Cypress.Commands.add('getByTestId', (selector, ...args) => {
  return cy.get(`[data-test=${selector}]`, ...args)
})

Cypress.Commands.add('getAllByTestId', (selector, ...args) => {
  return cy.get(`[data-test*=${selector}]`, ...args)
})
```

Reply:
That is outside the scope of this PR though